projects/gameboard-ui/src/app/api/game.service.ts

```diff
@@ -11,13 +11,11 @@ import { ChangedGame, Game, GameGroup, NewGame, SessionForecast, UploadedFile }
 import { TimeWindow } from './player-models';
 import { Spec } from './spec-models';
 
-@Injectable({
-  providedIn: 'root'
-})
+@Injectable({ providedIn: 'root' })
 export class GameService {
   url = '';
   private cache: CachedGame[] = [];
-  constructor(
+  constructor (
```

Comment:
Is a linter suggesting this?  Seems weird to put space between method name a param def.